instrumentation/kafka-streams/src/main/java/brave/kafka/streams/KafkaStreamsTracing.java

```diff
@@ -445,11 +491,50 @@ public <V, VR> ValueTransformerSupplier<V, VR> mapValues(String spanName,
       });
   }
 
+  /**
+   * Create a tracing-decorated {@link org.apache.kafka.streams.processor.api.ProcessorSupplier}
+   *
+   * <p>Simple example using Kafka Streams DSL:
+   * <pre>{@code
+   * StreamsBuilder builder = new StreamsBuilder();
+   * builder.stream(inputTopic)
+   *        .process(kafkaStreamsTracing.process("my-processor", myProcessorSupplier);
+   * }</pre>
+   *
+   * @see TracingKafkaClientSupplier
+   */
+  public <KIn, VIn, KOut, VOut> org.apache.kafka.streams.processor.api.ProcessorSupplier<KIn, VIn, KOut, VOut> process(String spanName,
+    org.apache.kafka.streams.processor.api.ProcessorSupplier<KIn, VIn, KOut, VOut> processorSupplier) {
+    return new NewTracingProcessorSupplier<>(this, spanName, processorSupplier);
+  }
+
+  /**
+   * Create a tracing-decorated {@link org.apache.kafka.streams.processor.api.FixedKeyProcessorSupplier}
+   *
+   * <p>Simple example using Kafka Streams DSL:
+   * <pre>{@code
+   * StreamsBuilder builder = new StreamsBuilder();
+   * builder.stream(inputTopic)
+   *        .processValues(kafkaStreamsTracing.processValues("my-processor", myFixedKeyProcessorSupplier);
+   * }</pre>
+   *
+   * @see TracingKafkaClientSupplier
+   */
+  public <KIn, VIn, VOut> org.apache.kafka.streams.processor.api.FixedKeyProcessorSupplier<KIn, VIn, VOut> processValues(String spanName,
+    org.apache.kafka.streams.processor.api.FixedKeyProcessorSupplier<KIn, VIn, VOut> processorSupplier) {
+    return new TracingFixedKeyProcessorSupplier<>(this, spanName, processorSupplier);
```

Comment:
So here it would be:
```suggestion
    return new TracingV2FixedKeyProcessorSupplier<>(this, spanName, processorSupplier);
```